### ISSUE TITLE
`bento check` should print tool installation gracefully

### ISSUE THREAD

**Issue by arashd** [returntocorp/bento#140]
Make sure you are running the latest version of Bento.

**Describe the bug**
I ran `bento check` on bento-core (a project where .bento.yml and .bento-whitelist.yml exist, but no node_modules exists because it's a python project.)

The tool installs eslint and I see output where the install logs of eslint are interspersed with the tool's output.

**Expected result**
I expected Bento to either: 
1. tell me that one of the tools I needed isn't installed before it started running the other two. (but I don't really know why I'd need eslit to be installed/run here), or
2. run the python tools alone.

**Actual result**
```bash
$ bento check
Running Bento checks...
  r2c.bandit: |               | 00:00, 🍜
  r2c.eslint: |               | 00:00, 🍜✔ Creating virtual environment for r2c.flake8
Installing {'eslint': Version('6.1.0'), 'eslint-config-airbnb': Version('18.0.1'), 'eslint-plugin-import': Version('2.18.2'), 'eslint-plugin-jsx-a11y': Version('6.2.3'), 'eslint-plugin-react': Version('7.14.3'), 'eslint-plugin-react-hooks': Version('1.7.0')}...       | 00:00, 🍜
✔ Python package bandit will be installedl resolveWithNewModule p-try@1.0.0 checking installable status
✔ Python package flake8 will be installedl resolveWithNewModule read-pkg@2.0.0 checking installable status
✔ Python package flake8-json will be installed
⸨            ░░░░░░⸩ ⠸ extract:es-abstrac⸨            ░░░░░░⸩ ⠋ extract:eslint-plugin-react: sill extract eslint-plugin-react@^7.14.3 extracted to /Users/arashdelijani/workspace/ret2co/bento-core/node_modules/.staging/eslint-plugin-react-⸨            ░░░░░░⸩ ⠹ extract:resolve: s⸨            ░░░░░░⸩ ⠸ extract:eslint-plugin-jsx-a11y: sill extract eslint-plugin-jsx-a11y@^6.2.3 extracted to /Users/arashdelijani/workspace/ret2co/bento-core/node_modules/.staging/eslint-plugin-j+ eslint-config-airbnb@18.0.1
+ eslint-plugin-jsx-a11y@6.2.3| 00:08, 🍤
+ eslint-plugin-import@2.18.2 | 00:09, 🍤
+ eslint-plugin-react@7.16.0
+ eslint@6.6.0
+ eslint-plugin-react-hooks@1.7.0
added 199 packages from 114 contributors and audited 609 packages in 9.26s
found 0 vulnerabilities



   ╭────────────────────────────────────────────────────────────────╮
   │                                                                │
   │       New minor version of npm available! 6.9.0 → 6.12.1       │
   │   Changelog: https://github.com/npm/cli/releases/tag/v6.12.1   │
   │               Run npm install -g npm to update!                │
   │                                                                │
   ╰────────────────────────────────────────────────────────────────╯

  r2c.bandit: |███████████████| 00:12, 🍱
  r2c.eslint: |███████████████| 00:11, 🍱
  r2c.flake8: |███████████████| 00:11, 🍱

tests/integration/react/index.jsx
  1:19 error   Unable to resolve path to module 'react'.     r2c.eslint     import/no-unresolved
  2:22 error   Unable to resolve path to module 'react-dom'. r2c.eslint     import/no-unresolved

⚠ 2 findings in 12.39 s

To suppress all findings run `bento archive`.
``` 
**Additional context**
$ bento --version
bento/0.3.0b1

**Comment by dlukeomalley** [comment]
I ran into this issue just now as well on release/0.3:

![image](https://user-images.githubusercontent.com/605906/67996199-8e139a80-fc0b-11e9-83a6-34ac13a34587.png)

**Comment by dlukeomalley** [comment]
This is addressed in our upcoming 0.7 release! CC @nbrahms
